Qualitätsanforderungen.md

```diff
@@ -1,12 +0,0 @@
-## Qualitätsanforderungen
```

Comment:
Diese Datei sollte erst gelöscht werden, wenn ihr Inhalt nach LaTeX konvertiert wurde. Also einfach den Kram hier in ne .tex-Datei packen und in der Projektdoku inputten bitte